src/App.js

```diff
@@ -8,7 +8,9 @@ const App = () => {
   const [nextTurn, setNextTurn] = useState(true);
 
   const handleGamePlay = (index) => {
-    
+    if (squares[index]) {
+      return;
```

Comment:
I am curious what this is returning?  This may work, but could become buggy in the future.